src/main/scala/chisel3/verilog/package.scala

```diff
@@ -0,0 +1,16 @@
+package chisel3
+
+import chisel3.stage.ChiselStage
+import firrtl.AnnotationSeq
+
+package object verilog {
+
+  implicit class ImplicitVerilog(module: => RawModule) {
+    def getVerilogString: String = ChiselStage.emitVerilog(module)
+    def emitVerilog(args: Array[String] = Array.empty,
+                    annotations: AnnotationSeq = Seq.empty) = {
+      (new ChiselStage).emitVerilog(module, args, annotations)
+      // TODO: this should probably not return anything, as we are writing a Verilog file
+    }
```

Comment:
I think you should either make it return the verilog `String` or `Unit`, but the return type needs to be defined

Reply:
This should be fixed by now.